lib/mockbuilder.py

```diff
@@ -187,5 +186,6 @@ def copy_results(self, package):
             package(Package): package whose result files will be copied
         """
         package_build_results_dir = os.path.join(
-            self.build_results_dir, package.name)
+            CONF.get('default').get('result_dir'), 'packages',
+            self.timestamp, package.name)
```

Comment:
This will require modifications in the infrastructure repository.
The archiving of artifacts in the `build_host_os` job and the copying of those archives in the `upload_build_artifacts`, `build_host_os_iso` and `upload_iso` jobs will need path changes. Maybe creating a `latest` symlink at the end of the build would make that easier?